assets/js/components/Dashboard/AlertsPage.tsx

```diff
@@ -171,61 +265,58 @@ const AlertsPage: ComponentType<Props> = (props: Props) => {
   };
 
   return (
-    <div
-      className={classNames("alerts-page", {
-        "alerts-page--hidden": !props.isVisible,
-      })}
-    >
-      <div className="page-content__header">Posted Alerts</div>
-      <div className="page-content__body">
-        <Container fluid>
-          <Row className="place-list__header-row">
-            <Col lg={3}>
-              <div
-                className="place-list__sort-label d-flex align-items-center"
-                onClick={sortLabelOnClick}
-                data-testid="sort-label"
-              >
-                {alertSortLabel}
-                {alertSortDirection === 0 ? <ArrowDown /> : <ArrowUp />}
-              </div>
-            </Col>
-            <Col lg={3} className="d-flex justify-content-end pe-3">
-              <FilterDropdown
-                list={MODES_AND_LINES}
-                onSelect={(value: any) => handleAlertModeOrLineSelect(value)}
-                selectedValue={alertModeLineFilterValue}
-                className="modes-and-lines"
-              />
-            </Col>
-            <Col lg={3} className="place-screen-types pe-3">
-              <FilterDropdown
-                list={SCREEN_TYPES}
-                onSelect={(value: any) => handleAlertScreenTypeSelect(value)}
-                selectedValue={alertScreenTypeFilterValue}
-                className="screen-types"
-              />
-            </Col>
-            <Col lg={3}>
-              <FilterDropdown
-                list={STATUSES}
-                onSelect={(value: any) => handleAlertStatusSelect(value)}
-                selectedValue={alertStatusFilterValue}
-                className="statuses"
-                disabled
-              />
-            </Col>
-          </Row>
-        </Container>
-        {filterAlerts()
-          .sort((a: Alert, b: Alert) =>
-            alertSortDirection === 0 ? compareAlerts(a, b) : compareAlerts(b, a)
-          )
-          .map((alert: Alert) => (
-            <AlertCard key={alert.id} alert={alert} />
-          ))}
-      </div>
-    </div>
+    <>
+      <Container fluid>
+        <Row className="place-list__header-row">
```

Comment:
I don't think you set these class names, but it looks like they might be wrong due to some copy/pasting. These should be `alert-list...`.

Reply:
Ah! https://github.com/mbta/screenplay/blame/alerts-feature/assets/js/components/Dashboard/AlertsPage.tsx#L182